general_rules/Procedure.tex

```diff
@@ -72,15 +72,9 @@ \subsection{Collisions}
 	\item \textbf{\iterm{Major collisions}:} If a robot crushes into something during a test, the robot is immediately stopped.	Additional penalties may apply.
 
 	\item \textbf{\iterm{Functional touching}:} Robots are allowed to apply pressure on objects, push away furniture and, in general, interact with the environment using structural parts other than their manipulators.
-	This is known as \iaterm{functional touching}.
+	This is known as \iterm{functional touching}.
 	However, the robot must clearly announce the collision-like interaction and kindly request not being stopped.\\
 	\textbf{Remark: } Referees can (and will) immediately stop a robot in case or suspicion of \emph{dangerous} behavior.
-
-	\item \textbf{Robot-Robot avoidance:} If two robots encounter each other, they both have to actively try to avoid the other robot.
-	\begin{enumerate}
-		\item A robot which is not going for a different route within a reasonable amount of time (e.g., \SI{30}{\second}) is removed.
-		\item A non-moving robot blocking the path of another robot for longer than a reasonable amount of time (e.g., \SI{30}{\second}) is removed.
-	\end{enumerate}
```

Comment:
I think this valid as the number of multi robot situations have been increasing

Reply:
Where? In none of the tests do we use multiple robots at once. The only times this may happen is during the setup days (in which case the robots do not follow these rules) or in the final (where the robots belong to the same team)

Reply:
I have seen situations in which two robots run the restaurant challenge at the same time. And multiple robots can be used in Open/Final challenges.

Reply:
In Open and Final challenges the robots belong to the same team. We do not need to specify how the robots should interact in that case since the team will opt for a working solution with two of their own robots.
As for the restaurant challenge, I have not seen it in 2019 or 2022. Don't know much about years before that. But it seems strange for that to be the only reason to include these rules. The restaurant also does not state that multiple robots may test simultaneously.

Reply:
I leave this one for you to discuss with other TC members.